frontend-ssr/components/cli_git/GitCli_CC.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {uuid} from '../../services/Utils';
```

Comment:
Make sure you fix all ESLint issues. For example, here there should be a space, like so `{ uuid }`. 

You should see some red lines underneath the code with problems. If you don't maybe it's not correctly configured
![image](https://user-images.githubusercontent.com/11327761/92312194-3e3df680-efc7-11ea-9ee8-517848d6a1af.png)

Reply:
Added the ESLint extension in VSCode.